client/layout/index.jsx

```diff
@@ -143,6 +143,7 @@ Layout = React.createClass( {
 		return (
 			<div className={ sectionClass }>
 				<DocumentHead />
+				<QuerySites allSites />
```

Comment:
I think the motivation was to be more preemptive in fetching sites data, despite not all sections requiring it. For now, I'd be inclined to leave it in `Layout`, but I don't feel strongly about either.